src/main/java/org/checkerframework/checker/crypto/CryptoAnnotatedTypeFactory.java

```diff
@@ -74,12 +78,73 @@ public boolean isSubtype(final AnnotationMirror subtype, final AnnotationMirror
 
         @Override
         public AnnotationMirror greatestLowerBound(AnnotationMirror a1, AnnotationMirror a2) {
-            return CryptoAnnotatedTypeFactory.this.BOTTOM;
+            if (AnnotationUtils.areSameByName(a1, BOTTOM)
+                    || AnnotationUtils.areSameByName(a2, BOTTOM)) {
+                return CryptoAnnotatedTypeFactory.this.BOTTOM;
```

Comment:
```suggestion
                return BOTTOM;
```

Reply:
It looks strange that this line uses the field differently than the lines before. Isn't it simpler like this?

Reply:
Yeah, fixed.